### PR TITLE
empty agents.conf error and export error

### PR DESCRIPTION
**Pull request by sortizowlh**
Hi team, this PR resolves:

- Error opening agents.conf from the edit group configuration button
- Error opening agents.conf from file actions
- Export to PDF

Closes: #2893

### CODE REVIEW

File: server/controllers/wazuh-reporting.ts [hunk 1]
```diff
@@ -1247,7 +1248,11 @@ export class WazuhReportingCtrl {
           log('reporting:createReportsGroups', error.message || error, 'debug');
 
         };
-        if (Object.keys(configuration.affected_items[0].config).length) {
+        
+        console.log("configuration.affected_items.length");
+        console.log(configuration.affected_items.length);
```
Comment (Desvelao) [inline server/controllers/wazuh-reporting.ts:1253]: Remove these `console.log` statements.

**Comment by Desvelao** [comment]
I tested the PR and it is working now. Good job!

**Comment by Desvelao** [comment]
You should add the change to `CHANGELOG.md` file.

**Review by pablomarga — APPROVED** [review]
LGTM!

File: server/controllers/wazuh-reporting.ts (hunk 1 above)
Comment (sortizowlh) [reply server/controllers/wazuh-reporting.ts]: Logs removed

**Pull request merged**